hibernate-core/src/main/java/org/hibernate/id/enhanced/SequenceStyleGenerator.java

```diff
@@ -341,9 +389,10 @@ protected Identifier determineValueColumnName(Properties params, JdbcEnvironment
 	 * Called during {@link #configure configuration}.
 	 *
 	 * @param params The params supplied in the generator config (plus some standard useful extras).
+	 *
 	 * @return The initial value
 	 */
-	@SuppressWarnings({"WeakerAccess"})
+	@SuppressWarnings({ "WeakerAccess" })
```

Comment:
Is your IDE set to auto-change lines?  Or did you manually make all of these kind of changes?

Reply:
I run a CTRL+ALT+L, so it applied these changes for all the code in this class.